README.md

```diff
@@ -26,18 +26,60 @@ Anonymous Zether is not yet feasible for use in the Ethereum mainnet (see the [t
 
 The [packages](packages) folder contains the Zether Smart Contract and auxiliary contracts, as well as scripts to interact with it from the `geth` console.
 
-### User Instructions
+### Quickstart 
 
-#### Setting things up
-- Spin up a Quorum cluster (e.g., follow the steps of [the 7nodes example](https://github.com/jpmorganchase/quorum-examples/tree/master/examples/7nodes)).
-- Start the Spring application inside the `/prover` directory of this repo. The easiest way to do this is to import the Maven project into an IDE, and execute `zkp/src/main/java/zkp/App.java`.
-- In the main directory, type `yarn`.
+Deploy the ZSC (Zether Smart Contract) to a running Quorum cluster and make some anonymous transfers.
+
+## Install Prerequisites
+* [Java](https://www.java.com/en/download/help/download_options.xml) tested version 11.0.2
+* [mvn](https://maven.apache.org/install.html) tested with version 3.5.3             
+* [Yarn](https://yarnpkg.com/en/docs/install#mac-stable) tested with version 0.1.0
+* [nodejs](https://nodejs.org/en/download/) tested with version v10.15.3
+
+## Deploy a Quorum Network
+
+* Spin up a Quorum cluster (e.g., follow the steps of [the 7nodes example](https://github.com/jpmorganchase/quorum-examples/tree/master/examples/7nodes)).
+* **note** for the nodejs example in this project to work, websockets need to be enabled when starting up geth/quorum:
+  websocket geth flags `--ws`, `--wsport 23000`, `--ws --wsorigins=*`. 
+
+## Run the Prover
+
+The prover is a Java Spring application located inside the [`prover`](prover) directory of this repo.
+
+```
+# install yarn
+$> brew install yarn
+# note: tested with version
+$> yarn --version
+1.17.0
+
+$> cd anonymous-zether/prover
+# build the project
+$> yarn
+# start the prover
+$> mvn spring-boot:run
+```
+
+* The prover should now be up and running on http://localhost:8080.
+
+## Run the nodejs demo
+
+The nodejs [example project](packages/example) in this repo, will deploy the necessary contracts:  [ZetherVerifier.sol](packages/protocol/contracts/ZetherVerifier.sol), 
+[BurnVerifier.sol](packages/protocol/contracts/BurnVerifier.sol), [CashToken.sol](packages/protocol/contracts/CashToken.sol), and finally 
+[ZCS.sol](packages/protocol/contracts/ZSC.sol) which is dependent on the previous contracts. 
+
+Once the ZCS contract is deployed, the nodejs application will fund the account, add some 'friends',
+and make an anonymous transfer. 
+
+**Requires** that geth is started with `ws`, `--wsport 23000`, `--ws --wsorigins=*` flags.
 
-An automated demo can then be run by typing
 ```bash
-node packages/example
+$> cd anonymous-zether
```

Comment:
ultra-minor point: above (e.g. line 56) it's assumed that we're already sitting in the `anonymous-zether` directory. would it be more consistent to maintain this assumption, i.e. to remove this line?